contracts/IColony.sol

```diff
@@ -275,7 +275,49 @@ contract IColony {
   /// @param _role Id of the role, as defined in `ColonyStorage` `MANAGER`, `EVALUATOR` and `WORKER` constants
   /// @param _token Address of the token, `0x0` value indicates Ether
   function claimPayout(uint256 _id, uint256 _role, address _token) public;
-  
+
+  /// @notice Start next reward payout in `_token`. All funds in that are in reward pot at the time will become unavailable.
+  /// All tokens will be locked, and can be unlocked by calling `waiveRewardPayout` or `claimRewardPayout`.
+  /// @param _token Addess of the token used for reward payout
+  function startNextRewardPayout(address _token) public returns (uint256);
+
+  /// @notice Claim the reward payout at `_payoutId`. User needs to provide his reputation and colony-wide reputation
```

Comment:
Just some notes on comments language:
- Pls use the non-gender specific `their` instead of `his` or `hers`. There may be women interacting with this contract you know :)